sidebars.json

```diff
@@ -7,16 +7,17 @@
         {
             "type": "category",
             "label": "Advanced",
-            "items": [
-                "advanced/writing-declarations",
-                "advanced/compiler-annotations",
-                "advanced/functions-and-the-self-parameter"
-            ]
+            "items": ["advanced/writing-declarations", "advanced/compiler-annotations"]
         },
         {
             "type": "category",
             "label": "API",
             "items": ["api/overview", "api/plugins", "api/printer"]
+        },
+        {
+            "type": "category",
+            "label": "FAQ",
```

Comment:
Do you think we'll have more FAQ in the future? If we don't, I think it could be a top-level page, probably after `"caveats"` in the sidebar

Reply:
I think we might, but good idea to keep it top level for now